portal/src/atoms/Popover.jsx

```diff
@@ -21,6 +21,7 @@ export const Popover = ({
   onOpenChange,
   position,
   alignment,
+  width = 'w-[30rem]',
```

Comment:
Let's name this prop `widthClass` to be more clear about what value it expects, and also add it to the JSDoc comment above.